app/src/main/java/edu/uc/groupProject/topten/MainActivity.kt

```diff
@@ -37,89 +23,6 @@ class MainActivity : AppCompatActivity() {
         }
 
         this.supportActionBar?.hide();
-        writeListToDatabase()
-        getUser()
         supportActionBar?.setDisplayShowTitleEnabled(false)
     }
-
-    /**
-     * Writes the list to the Firebase Database. Currently writes the mocked data below to the
-     * database.
-     */
-    fun writeListToDatabase(){
-
-        lateinit var firestore : FirebaseFirestore
-        firestore = FirebaseFirestore.getInstance()
-        firestore.firestoreSettings = FirebaseFirestoreSettings.Builder().build()
-
-        var testList = TestDTO("Top Fifteen Movies", false, true, true, Date())
-
-
-        var  listsReference = (firestore.collection("lists").document(testList.listName))
-
-
-        listsReference.set(testList).addOnSuccessListener {
-            Log.d("Firebase", "document saved");
-        }.addOnFailureListener{
-            Log.d("Firebase", "Save Failed");
-        }
-
-        var listItemCollectionReference = listsReference.collection("MyListItems")
-
-        var arrayOfListItemsToAdd : Array<ListItem> = createSampleListItems();
-
-        for(item in arrayOfListItemsToAdd){
-            listItemCollectionReference.document(item.title).set(item);
-        }
-
-    }
-
-    /**
-     * Creates the mocked data list used by the program for testing purposes
-     * @return sampleListItems
-     */
-    fun createSampleListItems():Array<ListItem>{
-
-        var sampleListItems = arrayOf(
-                ListItem("The Dark Knight", "A movie about Batman", 100),
-                ListItem("The Return of the King", "A movie about a ring and some eagles", 150),
-                ListItem("The Empire Strikes Back", "A movie about some light wands and parent issues", 200),
-                ListItem("The Godfather", "n/a", 24),
-                ListItem("The Avengers", "They all team up to fight bad guys", 231),
-                ListItem("Inception", "", 12),
-                ListItem("E.T", "", 124),
-                ListItem("The Matrix", "", 42)
-        )
-        return sampleListItems
-    }
-
-    fun getUser(){
```

Comment:
Instead of assigning user information to views here, it will be assigned in the MainFragment, by calling methods on MainViewModel (see similar comment on MainFragment).

Reply:
Yes.  Data should live in the ViewModel... that's what the ViewModel is for.   It's best practice to avoid putting data directly in the Activity or Fragment.